workflows/theiaeuk/wf_theiaeuk_illumina_pe.wdl

```diff
@@ -135,11 +135,15 @@ workflow theiaeuk_illumina_pe {
       #     samplename = samplename
       # }
       if(defined(qc_check_table)) {
-        call qc_check.qc_check as qc_check_task {
+        call qc_check.qc_check_phb as qc_check_task {
           input:
             qc_check_table = qc_check_table,
             expected_taxon = expected_taxon,
```

Comment:
Given my ramblings above RE TheiaCoV and expected taxon being set to `organism`, I think this may actually be the point of an unnecessary input: since `expected_taxon` is pulled from the workflow input, `expected_taxon` will always equal `expected_taxon`, right? This may be more relevant of a workflow input of the user-defined `expected_taxon` overrode the gambit-defined merlin-tag, forcing a specific route for taxon-specific analysis. Though, as is, I'm not sure I see the current function of this task input.

Reply:
The qc_check_task currently does override the gambit-defined merlin tag with the expected_taxon if an expected_taxon value is provided. I built that functionality in so that if a laboratory is sequencing a species that is not well represented in the GAMBIT db they could simply provide the known taxonomic assignment as an input. Now with the addition of qc_check to TheiaCoV, the expected_taxon input has multiple functions. Does that make sense?

Reply:
The expected taxon override in TheiaProk has actually be part of the task from the start, it's just not frequently used